lib/rucio/web/rest/webpy/v1/authentication.py

```diff
@@ -61,6 +68,10 @@
     '/ssh_challenge_token', 'SSHChallengeToken',
     '/saml', 'SAML',
     '/validate', 'Validate',
+    '/OIDC', 'OIDC',
```

Comment:
lowercase in URL ?

Reply:
OK, Done. Do you want lowercase also in function names ?